src/pcre2test.c

```diff
@@ -7182,14 +7182,14 @@ while ((c = *p++) != 0)
       /* We used to have "while (isxdigit(*(++pt)))" here, but it fails
       when isxdigit() is a macro that refers to its argument more than
       once. This is banned by the C Standard, but apparently happens in at
-      least one MacOS environment. */
+      least one macOS environment. */
 
       for (pt++; isxdigit(*pt); pt++)
         {
         if (++i == 9)
           fprintf(outfile, "** Too many hex digits in \\x{...} item; "
                            "using only the first eight.\n");
-        else c = c * 16 + tolower(*pt) - ((isdigit(*pt))? '0' : 'a' - 10);
+        else c = c * 16 + (tolower(*pt) - ((isdigit(*pt))? '0' : 'a' - 10));
```

Comment:
nitpick: remove the strange additional parenthesis around `isdigit()`?

Reply:
the number is UINT16, and you can add +1 if you worry about that

Reply:
not sure I understand what you mean by that, but it sure doesn't worry me, it is just distracting and makes me curious.

Reply:
Github added the comment to the wrong place somehow. It is a comment for above.